forecast_solar/__init__.py

```diff
@@ -124,12 +151,22 @@ async def estimate(self) -> Estimate:
 
         Returns:
             A Estimate object, with a estimated production forecast.
+
+        Raises:
+            ForecastSolarRequestError: There is something wrong with the
+                variables used in the request.
         """
         data = await self._request(
             f"estimate/{self.latitude}/{self.longitude}"
             f"/{self.declination}/{self.azimuth}/{self.kwp}",
             params={"time": "iso8601", "damping": str(self.damping)},
         )
+
+        if not data["result"]["watts"]:
```

Comment:
Do we need this? If the location is not supported, we get a 400:

```
forecast_solar.exceptions.ForecastSolarRequestError: Can't find location data for latitude = 0 and longitude = 0 (error 5)
```

Reply:
the 400 was too general in previous error reports and was not clear to the user.

Reply:
Removed it, the new error exception handling is better 😉